tests/unit/dom/select-test.js

```diff
@@ -0,0 +1,146 @@
+import { module, test } from 'qunit';
+import { select, setupContext, teardownContext } from '@ember/test-helpers';
+import { buildInstrumentedElement } from '../../helpers/events';
+import hasEmberVersion from '@ember/test-helpers/has-ember-version';
+import { isIE11 } from '../../helpers/browser-detect';
+
+let clickSteps = ['focus', 'focusin', 'input', 'change'];
+
+if (isIE11) {
+  clickSteps = ['focusin', 'input', 'change', 'focus'];
+}
+
+module('DOM Helper: select', function(hooks) {
+  if (!hasEmberVersion(2, 4)) {
+    return;
+  }
+
+  let context, element;
+
+  hooks.beforeEach(function() {
+    context = {};
+  });
+
+  hooks.afterEach(async function() {
+    element.setAttribute('data-skip-steps', true);
+    if (element) {
```

Comment:
how could this condition fail if we call a method on the `element` above?

Reply:
Fair point, this seems to be in all the unit test, happy to remove, but perhaps we should create a issue to remove this.